### DISCUSSION TITLE
How to access 'manual visit' functions using Vue 3's <script setup> syntax

### DISCUSSION THREAD

**Discussion by stevie-c91** [inertiajs/inertia#1011] [Help (Vue)] [answered]
Hi, 

Does anyone know how to access [these functions](https://inertiajs.com/manual-visits) when using [<script setup>](https://v3.vuejs.org/api/sfc-script-setup.html)?

The examples in the docs return undefined.

Thanks.

**Comment by ajnsn** [comment, marked as answer]
Hey @stevie-c91 

you need to import Inertia, see example below!

```vue
<script setup>
import { Inertia } from '@inertiajs/inertia';

function visitHelloWorld() {
  Inertia.visit('/hello-world');
}
</script>
```

Hope this helps!

**Reply by stevie-c91** [reply]
That's perfect thanks @ajnsn, I could have swore I tried that!
